dash/testing/wait.py

```diff
@@ -68,6 +68,23 @@ def __call__(self, driver):
             return False
 
 
+class contains_class:
+    def __init__(self, selector, classname):
+        self.selector = selector
+        self.classname = classname
+
+    def __call__(self, driver):
+        try:
+            elem = driver.find_element(By.CSS_SELECTOR, self.selector)
+            classname = elem.get_attribute("class")
+            logger.debug(
+                "contains class {%s} => expected %s", classname, self.classname
+            )
+            return self.classname in str(classname)
```

Comment:
```suggestion
            return self.classname in str(classname).split(" ")
```
So we don't match part of a longer class, only an exact match of one of the classes.

Reply:
Ahhh yes! Great idea